### ISSUE TITLE
UI improvement suggestion

### ISSUE THREAD

**Issue by wuyupengwoaini** [jaegertracing/jaeger-ui#387]
Now we use uri as the span name, which may cause the operationName drop-down option on the interface to be incomplete.

like this:
<img width="265" alt="jaeger-ui" src="https://user-images.githubusercontent.com/5652901/58785968-34398500-8619-11e9-8012-a5d1d08990c6.png">

I suggest that you can display the complete operation when the mouse is hovering over the selected operationName.

**Comment by openmohan** [comment]
Yes, that works. But the user will be happier if he can see the operation name content while scrolling without hovering.

I have a couple of other solutions to discuss.
1. Having horizontal scroll bar for operation name select box
2. Instead of vertically stacked form, should we go for horizontal form? 

@tiffon @everett980 Please let us know your thoughts. Thanks

**Comment by alanisaac** [comment]
I also have this issue, where I see operation names often exceed the length of the drop-down.  When they do, hovering for tooltips to distinguish actions with the same first N characters is inconvenient.  

A couple alternatives to 1 & 2:

3. could the drop-down exceed the width of the parent selection box?  EX: 

![image](https://user-images.githubusercontent.com/12620443/71295874-c140f300-234b-11ea-9ce7-86dd463dc86c.png)

4. could the "Find Traces" box be horizontally resizable?  This could give users the option of playing with the ratio of space for filtering and viewing traces.

**Comment by everett980** [comment]
> 4. could the "Find Traces" box be horizontally resizable? This could give users the option of playing with the ratio of space for filtering and viewing traces.

I like this approach as it mirrors our solution to the same problem on the gantt chart view.

**Comment by everett980** [comment]
> 3. could the drop-down exceed the width of the parent selection box? EX (forgive my MS paint skills):

Is also pretty good. We recently added a popover-based dropdown that would work: [`FilteredList`](https://github.com/jaegertracing/jaeger-ui/tree/master/packages/jaeger-ui/src/components/common/FilteredList). This has the added benefit of reusing a more feature rich dropdown that has better searching for dropdown entries.
